model-optimizer/extensions/back/compress_quantized_weights.py

```diff
@@ -118,7 +124,8 @@ def pattern(self):
         )
 
     @staticmethod
-    def quantize_data(fake_quantize: Node, dst_type: type):
+    def quantize_data(fake_quantize: Node, dst_type: type,
+                      quantized_type: type, mode: str):
```

Comment:
nit: merge to one line? We have a limit of 120 characters which we fit here well.

Reply:
Done